v2/api/compute/v1alpha1api20201201/virtual_machine_scale_sets__spec_arm_types_gen.go

```diff
@@ -192,6 +181,17 @@ type VirtualMachineScaleSets_Spec_Properties_VirtualMachineProfile_NetworkProfil
 	NetworkInterfaceConfigurations []VirtualMachineScaleSets_Spec_Properties_VirtualMachineProfile_NetworkProfile_NetworkInterfaceConfigurationsARM `json:"networkInterfaceConfigurations,omitempty"`
 }
 
+// Deprecated version of VirtualMachineScaleSets_Spec_Properties_VirtualMachineProfile_OsProfile. Use v1beta20201201.VirtualMachineScaleSets_Spec_Properties_VirtualMachineProfile_OsProfile instead
+type VirtualMachineScaleSets_Spec_Properties_VirtualMachineProfile_OsProfileARM struct {
```

Comment:
Just curious, do you know why the name of this changed?

Reply:
Usually it would mean that the type is no longer explicitly named but is anonymous. However, only the Swagger changed so I'm not entirely sure!